build.cake

```diff
@@ -88,7 +126,14 @@ Task("Build")
                 .WithTarget("Build")
                 .WithProperty("Configuration", configuration)
                 .SetVerbosity(Verbosity.Minimal));
-	    }
+        }
+
+        version = context.EnvironmentVariable("GitVersion_MajorMinorPatch");
+        var nugetVersion = context.EnvironmentVariable("GitVersion_NuGetVersion");
+        var preReleaseTag = context.EnvironmentVariable("GitVersion_PreReleaseTag");
+        var semVersion = context.EnvironmentVariable("GitVersion_LegacySemVerPadded");
```

Comment:
Yea, this is not done. I have not gotten GitVersion working correctly in Cake yet.